src/app/components/lineage-graph/lineage-graph.component.spec.ts

```diff
@@ -11,6 +11,11 @@ import { LineageGraphNodeData } from "src/app/dataset-view/additional-components
 import { TEST_AVATAR_URL } from "src/app/api/mock/auth.mock";
 import _ from "lodash";
 import AppValues from "src/app/common/app.values";
+import { MatIconModule } from "@angular/material/icon";
+import { DisplaySizeModule } from "src/app/common/pipes/display-size.module";
+import { DisplayTimeModule } from "../display-time/display-time.module";
+import { SharedTestModule } from "src/app/common/shared-test.module";
+import { mockGraphNode } from "src/app/dataset-view/additional-components/data-tabs.mock";
 
 describe("LineageGraphComponent", () => {
```

Comment:
Missing a test that would check that the values of dataset properties are visible as parts of the panel. Basically, compare elements content with dataset attributes.